README.md

```diff
@@ -99,16 +99,19 @@ chs-gov-uk-notify-integration-api/
-2. Start with a letter design - probably a PDF, and evolve HTML and CSS resources to reproduce the letter as a web document. Bear in mind that some of the formatting of these letters will be print media specific, so it is not until you are producing PDFs that you will know exactly what the resulting letter will look like.
-3. Place common resources in the `assets/templates/letters/common` directory and those resources specific to the letter you are adding in the appropriate client directory (for example in `assets/templates/letters/chips`).
-4. Set up context validation to ensure that the required variable values are present to be able to create, print and send valid letters out. Extend `TemplateContextValidator.VALID_CONTEXTS` with an additional entry to do so.
-5. Test the result (of course!). 
+1. Start with a letter design - probably a PDF, and evolve HTML and CSS resources to reproduce the letter as a web document. Bear in mind that some of the formatting of these letters will be print media specific, so it is not until you are producing PDFs that you will know exactly what the resulting letter will look like.
+1. Place common resources in the `assets/templates/letters/common` directory.
+1. Place those resources specific to the letter you are adding in the appropriate client directory. For example, if your letter type is `CSIDVDEFLET` and you are creating a new template `v2.0` for `chips`, resources will go to `assets/templates/letters/chips/CSIDVDEFLET/v2.0/`. 
+1. Set up context validation to ensure that the required variable values are present to be able to create, print and send valid letters out. Extend `TemplateContextValidator.VALID_CONTEXTS` with an additional entry to do so.
+1. Test the result (of course!). 
 
 ### Background
 
-Template resources for letters are to be grouped by the client service or app that originates the request to send a letter. In the first case we have developed so far, we are dealing with a "direction" letter request made by the CHIPS application. Its template resources thus reside in the `chips` letter assets directory.
+Template resources for letters are to be grouped by the client service or app that originates the request to send a letter. For example, if a letter request is made by the CHIPS application, its template resources will reside in the `chips` letter assets directory.
 
-We have a number of HTML and CSS resources starting with a "root" HTML template file named `directionLetter_v1.html` that brings in the other files to create the final template.  The `_v1` suffix tells us that these letters are versioned and we are currently working on version 1.
+Resources are then grouped by letter id and template id. For example, for a new template `v2.0` for the `CSIDVDEFLET` letter id, its template resources will reside in the `CSIDVDEFLET/v2.0` subdirectory.
+
```

Comment:
Reading this, I get the impression that what is referred to here as a `letter id` is actually a `letter type`, and what is called a `template id` is really a `template version` in effect?

Reply:
Yes, you are spot on! We are using the letter type id from chips for the letter id here. And the template id does in fact act as a template version - I know we removed this recently!

Reply:
We should try to use names for things that reflect their usages/concepts/purposes/responsibilities accurately if it is possible to do so.